src/blueprint-utils.coffee

```diff
@@ -58,4 +58,9 @@ blueprintUtils.rangesToLinesText = (ranges) ->
   return pos
 
 
-module.exports = blueprintUtils
```

Comment:
__To think about:__ The original approach is imo better because it allows you to stub/spy functions inside module.